descope/api/client.go

```diff
@@ -231,10 +232,11 @@ type mgmtEndpoints struct {
 	accessKeyActivate   string
 	accessKeyDelete     string
 
-	ssoSettings string
-	ssoMetadata string
-	ssoMapping  string
-	updateJWT   string
+	ssoSettings        string
+	ssoMetadata        string
+	ssoMapping         string
+	updateJWT          string
+	createEmbeddedLink string
```

Comment:
I think this should start with `userXXX` and appear at the end of the user functions to keep consistent

Reply:
done